### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by DChan0319**

### CODE REVIEW

**Comment by DChan0319** [comment]
Not sure why it says i have conflicting files to be automatically merged.

File: client/src/components/dashPage/workoutsTab.jsx [hunk 1]
```diff
@@ -47,11 +47,15 @@ class WorkoutsTab extends Component {
     );
   }
 
+  starRerenderfunction(){
```
Comment (zgallagher08) [inline client/src/components/dashPage/workoutsTab.jsx]: can you delete this empty function?

File: client/src/components/dashPage/workoutsTab.jsx [hunk 2]
```diff
@@ -47,11 +47,15 @@ class WorkoutsTab extends Component {
...
+
+  }
+
   render() {
     return this.state.activeListItem === 0 ? (
       <Fragment>
         <WorkoutListContainer />
-        <WorkoutItemContainer />
+        <WorkoutItemContainer starRerenderfunction={this.starRerenderfunction}/>
```
Comment (zgallagher08) [inline client/src/components/dashPage/workoutsTab.jsx]: also delete the empty prop from here

**Review by zgallagher08 — CHANGES_REQUESTED** [review]
Awesome. One very small change, just delete that function

**Review by zgallagher08 — APPROVED** [review]

**Review by jaba9531 — APPROVED** [review]
Nice work

**Review by amelendez37 — APPROVED** [review]

**Pull request merged**